corehq/apps/app_manager/static/app_manager/json/commcare-app-settings.yml

```diff
@@ -156,6 +156,9 @@
   values:
     - ['2.0', '2.0']
   default: '2.0'
+  disabled: true
+  disabled_txt: Oops! This setting has been discontinued. We recommend you change this setting to Version 2, and it will no longer appear on your settings page.
```

Comment:
Does this get translated at a later point when it gets shown in the UI? If not then it might be good to consider translating this text.

Reply:
Also, just a nitpick on the wording of the text. Perhaps it can be something like the following:
 `Oops! This setting has been discontinued. We recommend you change this setting to Version 2. Upon doing so this setting will no longer appear on this settings page.`

The above might just make it a bit more clear as to when the setting will stop appearing on the page. No strong feelings however, and fine to leave as is if you feel the current text is clear enough.

Reply:
I love the idea that these be translated, but I think that might be a significantly heavier lift. I think it's okay to continue doing what the rest of this file does.

It is also worth noting that since you copied and pasted the exact text from line 111 that Danny wrote in 2014 (!), if that text is already translated, then this text will be translated too

Reply:
Yes, I did use the exact same text. 
Just did a quick check and these messages in the yaml file are indeed translated.